selfdrive/ui/ui.hpp

```diff
@@ -19,12 +19,13 @@
 #include "nanovg.h"
 
 #include "common/mat.h"
-#include "common/visionipc.h"
 #include "common/visionimg.h"
 #include "common/framebuffer.h"
 #include "common/modeldata.h"
 #include "common/params.h"
 #include "sound.hpp"
+#include "visionipc.h"
+#include "visionipc_client.h"
```

Comment:
we should probably standardize our include path convention. just reading this header file, I have no idea where to find visionipc*.h

Reply:
We did the same for messaging, but I agree.